_posts/2018-06-26-why-you-need-https.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: "Why your website needs HTTPS"
+date: 2018-06-26
+author: Chris Russo
+tags:  marketing user-experience best-practices drupal drupal-planet
+summary: The upsides of HTTPS now greatly exceeds any previously valid tradeoffs. If you're not using it, it's time.
+featured_image: "/blog/https.jpg"
+featured_image_alt: "Image endorsing move to HTTPS from HTTP"
+drupal_planet_summary: |
+  There are many considerations for site owners on when to make the move to HTTPS. Google has just stepped up the pressure. Secure communication is especially important for Drupal site owners and the time to make the move to HTTPS is now.
+---
+
+For some time, major internet players have advocated the benefits of a ubiquitous, secure internet, touting its myriad benefits for all users and service providers. The most prominent and steadfast among them is Google. In the next week, continuing a multi-year effort to shepherd more traffic to the secure web, Google will make perhaps its boldest move to date which will negatively impact all organizations not securely serving their website over HTTPS.
+
+To quote the official [Google Security Blog](https://security.googleblog.com/2018/02/a-secure-web-is-here-to-stay.html)
+
+> Beginning in July 2018 with the release of Chrome 68, Chrome will mark all HTTP sites as “not secure”
+
+![Chrome insecure message for HTTP]({{ site.url }}/assets/img/blog/chrome-68-not-secure.png)<br>
+ <span class="caption"> Googleblog </span>
+
+<!-- Test center image - doesn't work <img src="/assets/img/blog/chrome-68-not-secure.png" alt="Chrome insecure message for HTTP" class="blog-image-center"> -->
+
+Given the ambiguous "in July 2018", with no clearly communicated release date for Chrome 68, it's wise to err on the side of caution and assume it will roll out on the 1st. We have readied our partners with this expectation.
+
+So what does this mean for your organization if your site is not served over HTTPS? In short, it's time to make the move. Let's dig in.
+
+## What is HTTPS?
+
+HTTP, or **H**yper**T**ext **T**ransfer **P**rotocol, is the internet technology used to communicate between your web browser and the servers that the websites you visit are on. HTTPS is the secure version (s for secure) which is served over TLS: Transport Layer Security. What these technical acronyms equate to are tools for internet communication that verify you're communicating with who you think you are, in the way you intended to, in which no one besides the intended recipient can understand. We'll touch on the specifics in a moment and why they're important. Put simply, HTTPS enables secure internet communication.
```

Comment:
Instead of "in which no one besides the intended recipient can understand", "in a way that no one besides the intended recipient can understand"